### PR TITLE
Migrated to node10 for AppCenterTestV1 task

### PR DESCRIPTION
**Pull request by AndreyIvanov42**
**Task name**: [Node 10] Migrated AppCenterTestV1 task

**Description**: Migrated to node10 for AppCenterTestV1 task.

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

**Comment by AndreyIvanov42** [comment]
Linked issue - https://github.com/microsoft/build-task-team/issues/2065

File: Tasks/AppCenterTestV1/task.loc.json [hunk 1]
```diff
@@ -12,9 +12,10 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 1,
-    "Minor": 198,
+    "Minor": 199,
```
Comment (anatolybolshakov) [inline Tasks/AppCenterTestV1/task.loc.json:15]: Please bump to 200

**Comment by anatolybolshakov** [comment]
I believe this task requires extended testing - since there are major updates for task-lib dependency.
Btw source code changes looks good, thanks!

**Review by anatolybolshakov — APPROVED** [review]

**Review by max-zaytsev — APPROVED** [review]

**Comment by AndreyIvanov42** [comment]
cc @geekzter for visibility

**Comment by ondrej-merkun-ms** [comment]
@AndreyIvanov42 Could you please test some basic flows to ensure that your changes work as expected, as per the PR checklist? 🙂
image

**Comment by AndreyIvanov42** [comment]
Hi @ondrej-merkun-ms 
I tried to create a pipeline to test changes. You can find a pipeline [here](https://github.com/AndreyIvanov42/AppCenter-Samples). But unfortunately, I am not so familiar with mobile development and AppCenter functionality and couldn't solve happened issues.
Possibly you can help me with this.

**Pull request closed without merging**